frappe/email/queue.py

```diff
@@ -348,17 +348,20 @@ def flush(from_test=False):
 				smtpserver = SMTPServer()
 				smtpserver_dict[email.sender] = smtpserver
 				
-			send_one_args = {
-				'email': email.name,
-				'smtpserver': smtpserver,
-				'auto_commit': auto_commit,
-				'from_test': from_test
-			}
-			enqueue(
-				method = 'frappe.email.queue.send_one',
-				queue = 'short',
-				**send_one_args
-			)
+			if from_test:
+				send_one(email.name, smtpserver, auto_commit, from_test=from_test)
+			else:
+				send_one_args = {
+					'email': email.name,
+					'smtpserver': smtpserver,
+					'auto_commit': auto_commit,
+					'from_test': from_test
```

Comment:
from test will always be false here, so its not needed i think?

Reply:
> from test will always be false here, so its not needed i think?

Yes also from_test is not at all used in the send_one function

Reply:
could you remove it please?